op-node/node/api.go

```diff
@@ -71,16 +71,17 @@ func (n *adminAPI) SequencerActive(ctx context.Context) (bool, error) {
 
 // PostUnsafePayload is a special API that allow posting an unsafe payload to the L2 derivation pipeline.
 // It should only be used by op-conductor for sequencer failover scenarios.
-func (n *adminAPI) PostUnsafePayload(ctx context.Context, payload *eth.ExecutionPayload) error {
+func (n *adminAPI) PostUnsafePayload(ctx context.Context, envelope *eth.ExecutionPayloadEnvelope) error {
 	recordDur := n.M.RecordRPCServerRequest("admin_postUnsafePayload")
 	defer recordDur()
 
-	if actual, ok := payload.CheckBlockHash(); !ok {
+	payload := envelope.ExecutionPayload
+	if actual, ok := envelope.CheckBlockHash(); !ok {
 		log.Error("payload has bad block hash", "bad_hash", payload.BlockHash.String(), "actual", actual.String())
 		return fmt.Errorf("payload has bad block hash: %s, actual block hash is: %s", payload.BlockHash.String(), actual.String())
 	}
 
-	return n.dr.OnUnsafeL2Payload(ctx, payload)
+	return n.dr.OnUnsafeL2Payload(ctx, envelope)
```

Comment:
Do you think we should make a V2 API for this instead of making this breaking change 

I'm leaning towards making the breaking change as it was added [recently](https://github.com/ethereum-optimism/optimism/pull/8800) and for op-ha which isn't used yet.

Reply:
If it's not used yet we should just make the breaking change. The envelope type will allow us to avoid breaking changes in the future.

Reply:
@0x00101010 can you confirm? I also think we can likely perform the breaking change here.

Reply:
Yes, it's fine. We can rework after this change is in. It'll be better to avoid future changes 👍